src/objective-c/tests/UnitTests/GRPCClientTests.m

```diff
@@ -104,6 +104,9 @@ @interface GRPCClientTests : XCTestCase
 @implementation GRPCClientTests
 
 + (void)setUp {
+  NSDictionary* environment = [[NSProcessInfo processInfo] environment];
```

Comment:
consider moving these logic into a separate utility function to extract the address.  This would potentially be shared across UnitTest & InteropTest that requires interop server address info.